examples/asr/conf/speech_multitask/fast-conformer_aed.yaml

```diff
@@ -7,6 +7,15 @@
 # One extra (linear projection) layer is added between FastConformer encoder and Transformer decoder if they have different hidden sizes
 # It is recommended to initialize FastConformer with ASR/SSL pre-trained encoder for better accuracy and faster convergence
 
+# Canary model family
```

Comment:
Can you add a sample training manifest json line to the top of the file

Reply:
Something like this?
`{"audio_filepath": "/path/to/audio/file.wav", "duration": 6.192, "text": "Text spoken in the audio.", "source_lang": "en", "target_lang": "en", "taskname": "asr", "pnc": "yes"}`

Reply:
yes